src/controllers/forgotPassword.js

```diff
@@ -0,0 +1,60 @@
+import  jwt from "jsonwebtoken";
```

Comment:
It would be better if you put your controllers in one folder since you have two files.